test/bigtest/tests/AcquisitionUnits/AcquisitionUnitsList/AcquisitionUnitsList-test.js

```diff
@@ -43,4 +44,17 @@ describe('Acquisition units list', () => {
       expect(acquisitionUnitEditor.isVisible).to.be.true;
     });
   });
+
+  describe('view acquisition unit deatils', () => {
+    const acquisitionUnitDetails = new AcquisitionUnitDetailsInteractor();
+
+    beforeEach(async function () {
+      await acquisitionUnitsList.units(0).click();
+      await acquisitionUnitDetails.whenLoaded();
+    });
+
+    it('should open unit details', () => {
```

Comment:
Test description is not so clear to me:
`Acquisition units list` `view acquisition unit deatils`  `should open unit details`

Reply:
is it related to auto focus?

Reply:
it's related to `onClick` prop

Reply:
I think Alex is saying about test of added autofocus